tests/integration/targets/vmware_rest_appliance/tasks/appliance_monitoring.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Get the list of the monitored items
+  vmware.vmware_rest.appliance_monitoring_info:
+  register: monitored_items
+
+- name: Assert monitored items are retrieved
+  ansible.builtin.assert:
+    that:
+      - monitored_items is defined
+      - monitored_items.value is not none
+      - monitored_items.value | length > 0
+    fail_msg: "Failed to retrieve the list of monitored items."
+
+- name: Print the monitored items
+  ansible.builtin.debug:
+    var: monitored_items
+
+- name: Query the monitoring backend for memory usage
+  vmware.vmware_rest.appliance_monitoring_query:
+    end_time: 2021-04-14T09:34:56.000Z
```

Comment:
Maybe try more recent dates. You've got an empty result.

```
    "monitoring_query_result": {
        "changed": false,
        "failed": false,
        "value": [
            {
                "data": [
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    "",
                    ""
                ],
                "end_time": "2021-04-14T09:34:56.000Z",
                "function": "AVG",
                "interval": "MINUTES5",
                "name": "mem.total",
                "start_time": "2021-04-14T08:34:56.000Z"
            }
```

Reply:
fixed